### DISCUSSION TITLE
Zim freezes on save, stays not-responsive for 1-3 seconds

### DISCUSSION THREAD

**Discussion by paapereira** [zim-desktop-wiki/zim-desktop-wiki#2068] [Q&A]
Zim 0.73.5, Portable, Windows 10

I'm having an issue that I already felt somewhat, but in the last days is getting very hard to work around it.

Basically, every time zim auto-saves (or Ctlr+S) the application freezes and stay non-responsive for some seconds.

I was in 073.4 with the same problem. I installed the new version 0.73.5 yesterday, index rebuilt, but I have the same problem.
In the first moments it seemed fine, but then it started happening again.

I don't know if it's my notebook size or number of notes/links?
1,55GB (including the .git folder, 2778 .txt files, 10411 total files (including the .git folder).

Can I do something to help you help me to figure this out?

The same notebook in Arch Linux, same version has zero issues so I'm thing it's the Windows version (or my machine?).

Thanks for the help. ( best note/task software ever :-) )

**Comment by jaap-karssenberg** [comment]
In theory zim should scale regardless of the number of notes. However it
has a limit for handling very large notes in a single page.

Do you any special setup for the filesystem layer - storing to a network
location or similar ?

-- Jaap

**Comment by paapereira** [comment]
It's a regular local disk. No network location involved.
My biggest .txt file is 68K.

I've been testing a theory. 
I deleted 'cache\zim', opened zim, waited for the index rebuild and all seems fine again. A little pause when auto-saving but not much.
A few days after, I had to did it again. I did it 3 times now.
My 'index.db' file is 4,98MB right now.

I also changed my location from 'C:\Users\.....' to 'X:' by 'Net Use X: \\Localhost\C$\Users\.....'. I had some errors opening attachments because the path was too long (Windows....).
